app/assets/stylesheets/items.scss

```diff
@@ -3,12 +3,31 @@
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
 .item-data {
-  color: white;
   display: flex;
-  justify-content: space-around;
-  width: 100%;
+  justify-content: space-between;
 
-  &.checked {
+  a {
+    color: white;
+  }
+}
+
+.checked {
```

Comment:
feels like it would make more sense with .item-name on the outside and .checked on the inside, just an observation though, since you could potentially have other styles on an item name

Reply:
I haven't changed this in #5 - I think it makes sense for the whole list item to be marked checked or not. Adding the strikethrough on the text was my quick solution, but it could also be a strike through the whole line, change of the background colour, etc. Does that make sense?